src/oaklib/converters/owl_to_fhir.py

```diff
@@ -0,0 +1,20 @@
+"""Convert OWL to FHIR CodeSystem JSON
+
+TODO's
+  1. Write functioning converter
+  2. Schema mappings (Obograph -> FHIR CodeSystem). SchemaAutomator? Other approach?
+  3. Add converter to CLI
+  4. Test(s)
+
+todo's (minor)
+  1. owl_to_fhir.py -> owl_to_fhir_codesystem.py?
```

Comment:
Rename `owl_to_fhir.py` -> `owl_to_fhir_codesystem.py`? Or is it implicit enough?

Reply:
I think explicit is good. Any part of FHIR can be encoded in RDF and thus OWL